### PR TITLE
Feat/argentinian format

### PR DESCRIPTION
**Pull request by KatuGT**
Beside the files needed for the new argentinian format included, I also added at the readme the MX and ES options that were missing

### CODE REVIEW

File: .husky/pre-commit [hunk 1]
```diff
@@ -1,4 +1 @@
-#!/bin/sh
-. "$(dirname "$0")/_/husky.sh"
-
```
Comment (mastermunj) [inline .husky/pre-commit:3]: Is this change required?

**Comment by mastermunj** [comment]
Hi @KatuGT , thank you for your contribution. Please check the comment I added on `.husky/pre-commit`.

File: .husky/pre-commit (hunk 1 above)
Comment (KatuGT) [reply .husky/pre-commit:3]: Hello ! you know, the terminal showsme this error when I try to add a commit while having that lines added again
image

The commit I am trying to end is: fix/added text to .husky pre commit

Comment (KatuGT) [reply .husky/pre-commit:3]: don´t mind me, I alreadi fixed it,

Comment (mastermunj) [reply .husky/pre-commit:3]: The screenshot indicates that for the commit you need to add type and subject.

You can remove the file from your commit as well.

> Hello ! you know, the terminal showsme this error when I try to add a commit while having that lines added again image
> 
> The commit I am trying to end is: fix/added text to .husky pre commit

Comment (KatuGT) [reply .husky/pre-commit:3]: @mastermunj  I already added again the lines, do you see there again?

Comment (mastermunj) [reply .husky/pre-commit:3]: @KatuGT You've added it at the end of the file, whereas it should be at the top.

Comment (KatuGT) [reply .husky/pre-commit]: @mastermunj you are aright, sorry. New fix sent, please check out egain when you have time

**Comment by mastermunj** [comment]
Thank you @KatuGT. Released the changes with v4.2.0
